Content.Server/GameObjects/Components/Mobs/MobComponent.cs

```diff
@@ -25,37 +34,65 @@ public class SpeciesComponent : SharedSpeciesComponent, IActionBlocker, IOnDamag
         private ThresholdType currentstate = ThresholdType.None;
 
         /// <summary>
-        /// Holds the damage template which controls the threshold and resistance settings for this species type
+        /// Holds the damage template which controls the threshold and resistance settings for this mob type
         /// </summary>
         private DamageTemplates DamageTemplate;
 
+        /// <summary>
+        /// Holds the body template which controls the organs, body functions and processes in living beings 
+        /// </summary>
+        public BodyTemplate Body { get; private set; }
```

Comment:
This should not be part of `Mob`. Make a separate component to store organs. Same goes for blood, make a separate component to store the blood vessel.